common/src/main/scala/com/intel/hibench/common/streaming/metrics/KafkaCollector.scala

```diff
@@ -60,9 +60,10 @@ class KafkaCollector(zkConnect: String, metricsTopic: String,
   }
 
   private def getPartitions(topic: String, zkConnect: String): Seq[Int] = {
-    val zkClient = new ZkClient(zkConnect, 6000, 6000, ZKStringSerializer)
+    val zkClient = new ZkClient(zkConnect, 6000, 6000, ZKStringSerializer$.MODULE$)
```

Comment:
This is scala code, why do we need use $.MODULE$ ?

Reply:
I do agree but if cod  is left as `ZKStringSerializer` shows following error :
```
error: object ZKStringSerializer in package utils cannot be accessed in package kafka.utils
```
since kafka_2.12 version now is 0.10.2.2 I could access to ZKStringSerializer as coded in this PR.

Reply:
I noticed `ZKStringSerializer` is a private object in 0.10.2.2.

Reply:
It is correct @carsonwang

Reply:
@carsonwang
For porting/migrating code to be compatible for newer version Hi Bench project might need other approach, so for this PR all changes in scala code reverted as it was and leave it untouched.